subsys/bluetooth/host/gatt.c

```diff
@@ -3904,6 +4177,9 @@ static int db_hash_commit(void)
 
 	BT_HEXDUMP_DBG(db_hash, sizeof(db_hash), "New Hash: ");
 
+	/* Indicate Service Changed */
```

Comment:
We should probably have a comment here why we are resetting the range, this is most likely going to happen after a firmware update where the database changes causing the hash to be different therefore all service changed data needs to be resetted.

Reply:
Done in last push!